### PR TITLE
Add ability to configure how often cluster is scaled up

### PR DESCRIPTION
**Pull request by kdelee**
I put in 5 seconds as a default, that may be aggressive...open to suggestions

### CODE REVIEW

File: ocp/cloudigrade/config-maps.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ data:
   aws-houndigrade-ecs-family-name: {{env "HOUNDIGRADE_ECS_FAMILY_NAME" | default .aws.houndigrade.ecs.familyName}}
   aws-houndigrade-availability-zone: {{env "HOUNDIGRADE_AWS_AVAILABILITY_ZONE" | default .aws.houndigrade.availabilityZone}}
   aws-houndigrade-autoscaling-group-name: {{env "HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME" | default .aws.houndigrade.autoscalingGroupName}}
+  aws-houndigrade-scale-up-cluster-schedule: "{{env "SCALE_UP_INSPECTION_CLUSTER_SCHEDULE" | default .aws.houndigrade.scaleUpClusterSchedule}}"
```
Comment (ghost) [inline ocp/cloudigrade/config-maps.yaml]: ```aws-houndigrade-scale-up-cluster-schedule: {{env "SCALE_UP_INSPECTION_CLUSTER_SCHEDULE" | default .aws.houndigrade.scaleUpClusterSchedule | toString}}```

File: ocp/test.yaml [hunk 2]
```diff
@@ -53,6 +53,7 @@ include:
             familyName: Houndigrade
           availabilityZone: us-east-1a
           autoscalingGroupName: EC2ContainerService-cloudigrade-test-us-east-1a-EcsInstanceAsg-N1MWE3Y628WK
+          scaleUpClusterSchedule: 5
```
Comment (ghost) [inline ocp/test.yaml]: The default is an hour, this should match that.

File: ocp/local.yaml [hunk 3]
```diff
@@ -50,6 +50,7 @@ include:
             familyName: Houndigrade
           availabilityZone: us-east-1a
           autoscalingGroupName:
+          scaleUpClusterSchedule: 5
```
Comment (ghost) [inline ocp/local.yaml]: Maybe 5 minutes instead of 5 seconds? Spamming a task that takes a few minutes to finish every 5 seconds seems a little too much.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Comment by kdelee** [comment]
:100: thanks :smile_cat:

**Pull request merged**